props/saber_fett263_buttons.h

```diff
@@ -4537,6 +4589,26 @@ SaberFett263Buttons() : PropBase() {}
         return true;
 #endif
 
+      case EVENTID(BUTTON_POWER, EVENT_CLICK_LONG, MODE_ON | BUTTON_AUX):
+        if (menu_) return true;
+        if (fusor.angle1() < - M_PI / 3) {
+          ChangeStyleNumberAllBlades(-1);
+        } else {
+          ChangeStyleNumberAllBlades(1);      
+        }
+        hybrid_font.PlayCommon(&SFX_ccchange);
+        return true;
+
+      case EVENTID(BUTTON_POWER, EVENT_CLICK_LONG, MODE_OFF | BUTTON_AUX):
+        if (menu_) return true;
+        num_fonts_ = RunCommandAndGetSingleLine("list_fonts", nullptr, 0, 0, 0);
```

Comment:
If you're going to call it every time anyways, why not just call it from inside ChangeFont?

Reply:
updated